ligo/src/treasuryDAO.mligo

```diff
@@ -97,6 +97,9 @@ let treasury_DAO_decision_lambda (proposal, store : proposal * storage) : return
     ([%Michelson ({| { FAILWITH } |} : string * unit -> return)]
         ("FAIL_DECISION_LAMBDA", ()) : return)
 
+let default_entrypoint (params, full_storage : bytes * full_storage): return_with_full_storage =
```

Comment:
AFAIU this exists only to be able to accept tez in transactions, am I correct?

If that's the case I suggest:
1. adding a comment to this function
2. renaming it to something more clear (e.g. `receive_xtz_entrypoint`), after all it now has little to do with `default`
3. change the name that it's saved with accordingly, instead of `default` (e.g. `receive_xtz`)

Reply:
Yeah that's right. In Lorentz, this is done via the default entrypoint, since we can't do that in LIGO, it makes sense to rename this properly. I've updated this change now, you can check it.

Reply:
Ok cool, still please add a comment to it.

The name is now clear, but I'd still like a sentence to say why we need it in the first place, aka to receive `xtz` because this is not allowed in most standard entrypoints.